apps/ensadmin/src/components/recent-registrations/components.tsx

```diff
@@ -235,10 +237,7 @@ export function RecentRegistrations() {
                     <TableCell>
                       {indexedChainId ? (
                         <ENSName
-                          address={getTrueOwner(
-                            registration.domain.owner,
-                            registration.domain.wrappedOwner,
-                          )}
+                          address={getTrueOwner(registration)}
```

Comment:
```suggestion
                          address={registration.owner}
```

Please see my related comment above for more details. When we get all this code right, this suggestion shows how the final result should look here.